mapping/sustainability.yaml

```diff
@@ -507,14 +507,30 @@
         grant
         equity
         guarantees
-    mapping: 'Project level: For each instrument, add a `finance` object to the `budget.finance` array. Set the `.id` appropriately. Set the `.financeCategory` to the matching code from the financeCategory codelist.'
+    mapping: |-
+        1. Get the `Finance` object in `.budget.finance` or `.contractingProcesses.summary.finance` that represents the financing arrangement. If none exists yet, [add a financing arrangement](../common.md#add-a-financing-arrangement).
+        2. Set the the financing arrangement's properties according to the instrument type:
+
+        Instrument type | `.assetClass` | `.type`
+        -- | -- | --
+        loan | 'debt' | 'loan'
+        grant | | 'grant'
+        equity | 'equity' | 'shares.listed' or 'shares.unlisted'
+        guarantee | | 'guarantee'
+
+        3. If the instrument is a concessional loan, set `.concessional` to `true`.
+        4. If the instrument is a non-concessional loan, set `.concessional` to `false`.
+        5. If the instrument is results-based, set `.resultsBased` to `true`.
     example: |-
         {
           "budget": {
             "finance": [
               {
                 "id": "1",
-                "financeCategory": "seniorDebt",
+                "assetClass": [
```

Comment:
Should the example not include everything mentioned in the mapping, `.resultsBased` is missing?

Reply:
Done in 361e438eee7641c489b9f512be1510594d156ca9